### ISSUE TITLE
Purpose of disable_auto_retries

### ISSUE THREAD

**Issue by kumar-dhanagopal** [oracle-terraform-modules/terraform-oci-base#7]
@hyder , Hi Ali, what's the purpose of disable_auto_retries ? I don't see it used anywhere in the code.

**Comment by hyder** [comment]
@briangustafson commented that "disable_auto_retries should always be false. (This variable made sense in some cases a very long time ago before the OCI provider improved its retry logic.) "

As a result, you'll see that in https://github.com/oracle-terraform-modules/terraform-oci-base/blob/master/docs/quickstart.adoc#instructions (Step 5) when the provider is created, this is hardcoded to false.

You'll also see that the variable has been removed from https://github.com/oracle-terraform-modules/terraform-oci-base/blob/master/variables.tf

We just need to remove it from the terraform options doc.

**Comment by kumar-dhanagopal** [comment]
Makes sense! I guess we should remove `disable_auto_retries` from `terraform.tfvars.example` as well then. I'll submit a PR.

**Comment by kumar-dhanagopal** [comment]
```
$ grep -r 'disable_auto_retries' .
./terraform.tfvars:  disable_auto_retries = false
./docs/terraformoptions.adoc:|disable_auto_retries
./docs/quickstart.adoc:  disable_auto_retries = false
./examples/gitlab/README.md:  disable_auto_retries = false
Binary file ./.terraform/plugins/linux_amd64/terraform-provider-oci_v3.47.0_x4 matches
./provider.tf:  disable_auto_retries = false
./terraform.tfvars.example:  disable_auto_retries = false
```

**Comment by hyder** [comment]
fixed in #10
